assets/js/util/tracking/createTrackEvent.js

```diff
@@ -56,10 +57,11 @@ export default function createTrackEvent(
 			event_label: label,
 			value,
 			dimension1: referenceSiteURL,
-			dimension2: isFirstAdmin ? 'true' : 'false',
+			dimension2: currentUserRoles.join( ',' ),
 			dimension3: userIDHash,
 			dimension4: global.GOOGLESITEKIT_VERSION || '',
 			dimension5: Array.from( enabledFeatures ).join( ', ' ),
+			dimension6: activeModules.join( ',' ),
```

Comment:
@kuasha420 A bit of a nit-pick, but can we align on a consistent way of concatenating comma-separated values here? Right now it's mixed between `', '` and `','` (without space). I don't have a strong preference, I just think it'd be better for it to be consistent. @aaemnnosttv @tofumatt @eugene-manuilov Let me know if you have any strong feelings towards one or the other.

Reply:
@eugene-manuilov can elaborate on this better, initially I went with the `', '`, matching `dimension5`.

Reply:
Ah, I see now. @eugene-manuilov In that case, how about we use only the comma throughout?

Reply:
@felixarntz, yes, just commas is better because this format is intended for machine-reading whilst `, ` is what we use in UI. @kuasha420 can you create a follow-up PR that updates the `dimension5` value as well?

Reply:
@eugene-manuilov @felixarntz I will create a follow up PR against `main`. Thanks for the clarification!

Reply:
+1 to omitting the added space. FWIW this is how `Array.toString` formats the output as well 👍